src/patternfly/components/Tooltip/docs/code.md

```diff
@@ -0,0 +1,23 @@
+## Overview
+
+A tooltip is used to provide contextual information for another component on hover.  The tooltip itself is made up of two elements: arrow and content.  The tooltip (and parent) can be wrapped in the overlay component for positioning based off of the parent.  For libraries that absolutely position off of the body element in the DOM, just the tooltip component can be used.  One of the directional modifiers (`.pf-m-left`, `.pf-m-top`, etc.) is required on the tooltip component (and on the overlay component's container element, if it is being used).
+
+## TODO: Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `role` or `aria` | `pf-c-tooltip` |  accessibility notes. |
+
+
+## Usage
+
+| Class | Applied To | Outcome |
+| -- | -- | -- |
+| `.pf-c-tooltip` | `<div>` |  Creates a tooltip. |
+| `.pf-c-tooltip__arrow` | `.pf-c-tooltip` |  Creates an arrow pointing towards the element the tooltip describes. ** Required.** |
+| `.pf-c-tooltip__content` | `.pf-c-tooltip` |  Creates the body of the tooltip. ** Required.** |
+| `.pf-m-left` | `.pf-c-tooltip` and `.pf-c-overlay` | (Default) - Puts arrow to the left of the tooltip's content. |
```

Comment:
why `pf-c-overlay` ?

Reply:
@andresgalante  - what should the developer do here specifically?  would you recommend a different approach?